changelog.md

```diff
@@ -45,8 +45,10 @@
 - [FIXED] scope method syntax loses parameters when used multiple times [#7058](https://github.com/sequelize/sequelize/issues/7058)
 - [INTERNAL] Updated to `generic-pool@3.1.6` [#7109](https://github.com/sequelize/sequelize/issues/7109)
 - [FIXED] findAll throws error if attributes option is formatted incorrectly [#7162](https://github.com/sequelize/sequelize/issues/7163)
+- [FIXED] model validation instance method now runs hooks by default [#7182](https://github.com/sequelize/sequelize/pull/7182)
```

Comment:
Link to issue #7178 , not PR

Changelog should say, "`Instance.validate` now run hooks by default", this is a breaking change, Need to add to bc breaks as well